packages/engine/src/credentials/CredentialSystem.ts

```diff
@@ -0,0 +1,73 @@
+import { NotificationService } from '@xrengine/client-core/src/common/services/NotificationService'
```

Comment:
engine cannot be importing client-core, the action should be processed in a separate system in client-core